region/maridia/inner-pink/Below Botwoon Energy Tank.json

```diff
@@ -126,6 +126,25 @@
         }
       }
     },
+    {
+      "link": [1, 1],
+      "name": "Leave With Runway - Frozen Zoa",
+      "requires": [
+        "h_canFrozenEnemyRunway"
+      ],
+      "exitCondition": {
+        "leaveWithRunway": {
+          "length": 5,
+          "openEnd": 0
+        }
+      },
+      "note": [
+        "The Zoa needs to be at least 3 pixels higher than when it is frozen at standing height.",
+        "It may help to freeze one while standing to the left of it before it starts moving horizontally,",
+        "then move to the slope to the right of it to position it. Note that it will move up 1 more pixel once it thaws.",
```

Comment:
```suggestion
        "It may help to freeze one while standing to the left of it before it starts moving horizontally,",
        "then move to a specific point on the slope to the right of it to position it vertically.",
```